README.md

```diff
@@ -1,12 +1,75 @@
 # SQL Project
 
 The purpose of this project is to use SQL as an exploratory tool, towards this end we will create a database with it's 
-corresponding tables and import datasets from [ourworldindata.org](www.ourworldindata.org). Finally we'll analyse this 
+corresponding tables and import datasets from [ourworldindata.org](http://www.ourworldindata.org). Finally we'll analyse this 
 information using SQL.
 
-This project was developed following the steps detailed below.
+This project is divided in two parts:
 
-## 1 - Identify the data requirements
+1. Creation of the Database and tables. Data population in the tables.
```

Comment:
Instead of `Data population in the tables` you can say `Load data into the database` or `Load datasets into the tables`